src/cli.rs

```diff
@@ -72,10 +73,16 @@ pub fn get_args() -> Args {
                  .help("Send SIGKILL to child processes (deprecated, use -s SIGKILL instead)")
                  .short("k")
                  .long("kill"))
-        .arg(Arg::with_name("debug")
-                 .help("Print debugging messages to stderr")
+        .arg(Arg::with_name("debounce")
+                 .help("Set the timeout between detected change and command execution, defaults to 500")
```

Comment:
Let's mark that as 500ms in the arg help.

Reply:
You got it